inventory/byo/hosts.localhost

```diff
@@ -0,0 +1,21 @@
+#bare minimum hostfile
+
+[OSEv3:children]
+masters
+nodes
+etcd
+
+[OSEv3:vars]
+ansible_ssh_user=root
```

Comment:
Change to `ansible_user`.  The `ssh` parts are deprecated.
http://docs.ansible.com/ansible/latest/intro_inventory.html